src/Smp.Web/Controllers/RequestsController.cs

```diff
@@ -82,6 +82,20 @@ public async Task<IActionResult> AcceptRequest(Guid userId, Guid senderId, byte
             return Ok();
         }
 
+        [HttpGet("[action]/{receiverId:Guid}/{requestTypeId:int}/{senderId:Guid}/"), Authorize]
+        public async Task<IActionResult> DeclineRequest(Guid receiverId, Guid senderId, byte requestTypeId)
+        {
+            var tkn = Request.Headers["Authorization"];
+            if (!_authService.AuthorizeSelf(tkn, receiverId)) return Unauthorized();
+
+            var request = await _requestsRepository.GetRequestByUserIdsAndType(receiverId, senderId, (RequestType) requestTypeId);
+            if (request == null) return BadRequest(new Error("invalid_request", "There is no request to accept."));
```

Comment:
It kind of implied that it's an `AcceptRequest` method just based on this string. I think the following better expresses the error.

```suggestion
            if (request == null) return BadRequest(new Error("invalid_request", "There is no pending request."));
```

Reply:
Funnily enough, that message was in the DeclineRequest method. I've changed the message to the suggested one, though.